android/src/main/java/com/cmcewen/blurview/BlurViewPackage.java

```diff
@@ -20,11 +20,13 @@
         return modules;
     }
 
-    @Override
-    public List<Class<? extends JavaScriptModule>> createJSModules() {
-        return Collections.emptyList();
-    }
-
+   // if react native < 47, open code
```

Comment:
Can you be so kind to move this comment under multiline comment block? /* ... */?